tests/Behat/MinkContext.php

```diff
@@ -100,4 +150,32 @@ protected function getLanguageSwitcherOverlay(): NodeElement {
     return $this->getSession()->getPage()->find('css', $selector);
   }
 
+  /**
+   * Get breadcrumb element.
+   *
+   * @return \Behat\Mink\Element\NodeElement
+   *   The breadcrumb element.
+   *
+   * @throws \Behat\Mink\Exception\ElementNotFoundException
+   */
+  protected function getBreadcrumb(): NodeElement {
+    $selector = 'div.ecl-page-header nav.ecl-breadcrumb';
+    $this->assertSession()->elementExists('css', $selector);
```

Comment:
This method returns the node element already, you can avoid the code duplication with the next return statement.

Reply:
Still to be adressed.